src/transformers/models/grounding_dino/modeling_grounding_dino.py

```diff
@@ -2762,37 +2730,61 @@ def __init__(self, matcher, num_classes, focal_alpha, losses):
         self.focal_alpha = focal_alpha
         self.losses = losses
 
-    # removed logging parameter, which was part of the original implementation
+    # Ignore copy
     def loss_labels(self, outputs, targets, indices, num_boxes):
         """
         Classification loss (Binary focal loss) targets dicts must contain the key "class_labels" containing a tensor
         of dim [nb_target_boxes]
         """
         if "logits" not in outputs:
             raise KeyError("No logits were found in the outputs")
+        if "one_hot_labels" not in outputs:
+            raise KeyError("No one_hot_labels were found in the outputs")
+        if "text_mask" not in outputs:
```

Comment:
I just tested it out and the outputs dict does not return a text_mask key. Is it the same as the label_maps thing?